Server/mods/deathmatch/logic/CColShape.cpp

```diff
@@ -57,6 +57,16 @@ void CColShape::SetPosition(const CVector& vecPosition)
     CStaticFunctionDefinitions::RefreshColShapeColliders(this);
 }
 
+void CColShape::AttachTo(CElement* pElement)
+{
+    CElement::AttachTo(pElement);
+    if (m_pAttachedTo)

```

Comment:
I believe the following change will resolve the crash I mentioned (the same in CClientColShape.cpp)
```cpp
if (pElement && m_pAttachedTo)
```

The reason being is that the destructor passes `NULL` to the `AttachTo` method, so we'd be trying to call `GetPosition` on `NULL`. I'll confirm this later on when I'm home.